scripts/binary/binary-cleanup.js

```diff
@@ -51,6 +51,7 @@ const getDependencyPathsToKeep = async (buildAppDir) => {
     'node_modules/html-webpack-plugin-5/index.js',
     'node_modules/mocha-7.2.0/index.js',
     'packages/server/node_modules/webdriver/build/index.js',
+    'packages/server/node_modules/@wdio/utils/build/node.js',
```

Comment:
See this thread: https://cypressio.slack.com/archives/C06G5D5KFCJ/p1746724397061149